### PR TITLE
fix: add includes of ignore one for words

### PR DESCRIPTION
**Pull request by SimonChaumet**
ignoreOneForWords content was ignored, now it is included and tested for french

### CODE REVIEW

**Comment by mastermunj** [comment]
Hi @SimonChaumet, for some reason Github action isn't running on this PR. Can you please help update the PR so that the checks can run?

**Comment by SimonChaumet** [comment]
@mastermunj is it good now ? Github says that workflows needs approval of maintainer

**Review by mastermunj — APPROVED** [review]

**Pull request merged**
